setup.py

```diff
@@ -27,7 +27,7 @@
 README_contents = open(os.path.join(current_dir, 'README.md'),
                        encoding='utf-8').read()
 doclines = README_contents.split("\n")
-dependencies = ['ftfy >= 4', 'msgpack-python', 'langcodes >= 1.4', 'regex >= 2015']
+dependencies = ['ftfy >= 5', 'msgpack-python', 'langcodes >= 1.4', 'regex == 2017.06.23']
```

Comment:
The motivation is to prevent inconsistency. Suppose Unicode 11 makes changes that have implications for word breaks. When `regex` upgrades to Unicode 11 word breaks, I don't want that to change the ConceptNet data until it's time to rebuild from scratch using Unicode 11.

The problem with date-based version numbers here is that I can't rely on any semver-like promises.